### PR TITLE
Remove beta guard for LG-278 and update FATS as necessary

### PR DESCRIPTION
**Pull request by Channyboy**
#build
Removes beta guard for #1598

### CODE REVIEW

**Comment by Channyboy** [comment]
#build
#libby

**Comment by Channyboy** [comment]
#build

**Comment by Channyboy** [comment]
#build #libby

File: dev/com.ibm.ws.logging.osgi/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -34,7 +34,7 @@ log.directory.desc=You can use this attribute to set a directory for all log fil
 message.file.name=Message file name
 message.file.name.desc=Name of the file to which message output is written relative to the configured log directory. The default value is messages.log. This file always exists and contains INFO and other (AUDIT, WARNING, ERROR, FAILURE) messages, in addition to System.out and System.err. This log also contains time stamps and the issuing thread ID. If the log file is rolled over, the names of earlier log files have the format messages_timestamp.log. Avoid trouble: Use the com.ibm.ws.logging.message.file.name property, except in cases where you are trying to change the configuration dynamically after startup.
 
-stackJoin=Format stack traces as a single event in logs.
+stackJoin=Format stack traces as a single event in logs
```
Comment (dmuelle) [inline dev/com.ibm.ws.logging.osgi/resources/OSGI-INF/l10n/metatype.properties:37]: I think this does get a period. It reads kind of like a fragment, but it's a complete imperative sentence. If we're going to use a period after the desc at line 38, we should use one here too

Comment (Channyboy) [reply dev/com.ibm.ws.logging.osgi/resources/OSGI-INF/l10n/metatype.properties:37]: @dmuelle this attribute/label isn't allowed a period according to the ant_schema validator unittests :(

Comment (dmuelle) [reply dev/com.ibm.ws.logging.osgi/resources/OSGI-INF/l10n/metatype.properties:37]: ah ok- nevermind! LGTM

**Review by dmuelle — APPROVED** [review]

File: dev/com.ibm.ws.logging.osgi/src/com/ibm/ws/logging/internal/osgi/stackjoiner/StackJoinerManager.java [hunk 2]
```diff
@@ -211,11 +211,17 @@ public synchronized void resolveStackJoinFeature(Map<String, Object> serverConfi
...
-            } else if (((Boolean) serverConfigMap.get(STACK_JOIN_SERVER_XML_CONFIG_NAME)) == false){
-                isStackJoinEnabled = false;
+            
+            Object o;
+            if ((o = serverConfigMap.get(STACK_JOIN_SERVER_XML_CONFIG_NAME)) != null) {
+                if (((Boolean) o) == true) {
+                    isStackJoinEnabled = true;
+                } else if (((Boolean) o) == false){
+                    isStackJoinEnabled = false;
+                }
```
Comment (fmhwong) [inline dev/com.ibm.ws.logging.osgi/src/com/ibm/ws/logging/internal/osgi/stackjoiner/StackJoinerManager.java]: Can the above be simplified to
`isStackJoinEnabled = (Boolean) o`

**Review by fmhwong — APPROVED** [review]
LGTM

**Comment by Channyboy** [comment]
#build #libby

**Comment by Channyboy** [comment]
Failures not related to this PR (UI and JCA)

**Pull request merged**